x-pack/plugins/apm/server/lib/errors/get_error_group.ts

```diff
@@ -69,9 +68,9 @@ export async function getErrorGroup({
   };
 
   const resp = await client<APMError>('search', params);
-  const error = oc(resp).hits.hits[0]._source();
-  const transactionId = oc(error).transaction.id();
-  const traceId = oc(error).trace.id();
+  const error = resp.hits.hits[0]._source;
+  const transactionId = error.transaction.id;
+  const traceId = error.trace.id;
```

Comment:
Don't you need `idx` here? Say, if there are 0 results, hits will be `[]` and accessing `hits[0]._source` will throw a null-pointer, right?